### PR TITLE
UI :  default values to issuer and signatory 

### PR DESCRIPTION
**Pull request by aishwa8141**

### CODE REVIEW

File: certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfSignature.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import java.security.KeyStore;
+import java.security.PrivateKey;
+import java.security.Security;
+import java.security.cert.Certificate;
+
+public class PdfSignature {
+
+    private static Logger logger = LoggerFactory.getLogger(PdfConverter.class);
+
+    private String keyStoreName = "conf/certificate.pfx";
```
Comment (indrajra) [inline certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfSignature.java]: Must be picked from env

File: certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfSignature.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            Security.addProvider(provider);
+
+            KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+            keyStore.load(new FileInputStream(keyStoreName), password);
+            String alias = keyStore.aliases().nextElement();
+            PrivateKey privateKey = (PrivateKey) keyStore.getKey(alias, password);
+            Certificate[] chain = keyStore.getCertificateChain(alias);
+
+            PdfSignatureAppearance appearance = stamper.getSignatureAppearance();
+            appearance.setReason("Digital signature");
```
Comment (indrajra) [inline certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfSignature.java]: no way these can come up!

File: ui-sample/angular-ui/src/assets/jsons/certificate.json [hunk 3]
```diff
@@ -137,6 +138,7 @@
                 "required": true,
                 "displayProperty": "Editable",
                 "visible": true,
+                "value":"https://gcert.gujarat.gov.in/gcert/",
```
Comment (indrajra) [inline ui-sample/angular-ui/src/assets/jsons/certificate.json]: please dont use these. Use some other trivial url.

File: ui-sample/angular-ui/src/assets/jsons/signatory.json [hunk 4]
```diff
@@ -17,6 +17,7 @@
                 "required": true,
                 "displayProperty": "Editable",
                 "visible": true,
+                "value":"CEO Gujarat",
```
Comment (indrajra) [inline ui-sample/angular-ui/src/assets/jsons/signatory.json]: no mention of states here.

**Comment by aishwa8141** [comment]
PR 55 is split into 56 (default view) and 57(pdf signature)

**Pull request closed without merging**